tron/commands/client.py

```diff
@@ -183,7 +183,7 @@ def request(self, url, data=None):
         uri = urllib.parse.urljoin(self.url_base, url)
         response = request(uri, data)
         if response.error:
-            raise RequestError("%s: %s" % (uri, response))
+            raise RequestError("%s: %s" % (uri, response), response)
```

Comment:
This looks a little funny to me because there is the tuple that gets parsed by the string, and then there is a second argument to the exception? 

fstrings again I think will make this more clear, because the response object will be turned into a string, right?

Reply:
How about returning RequestError(response.content), since response.content is the error message you care about? Then tronview can just print the RequestError instead of inspecting it.

Reply:
Yeah, I was thinking the same thing @qui.